onto_spread_ed/templates/base.html

```diff
@@ -24,12 +24,12 @@
               integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
               crossorigin="anonymous">
         <!-- Bootstrap filter table CSS -->
-        <link rel="stylesheet" href="/static/excel-bootstrap-table-filter-style.css">
+        <link rel="stylesheet" href="{{ config['URL_PREFIX'] }}/static/excel-bootstrap-table-filter-style.css">
```

Comment:
Is there a reason you didn't use `url_for` here?

Reply:
Initially when we were changing the prefixes, all the href in html files were changed in this following way itself and it worked and for those which didnt work properly we added the url_for. 
So most of the hrefs are configured similarly.